src/main/java/hudson/plugins/analysis/core/PluginDescriptor.java

```diff
@@ -212,7 +203,12 @@ public FormValidation doCheckDefaultEncoding(@QueryParameter final String defaul
      */
     public FormValidation doCheckPattern(@AncestorInPath final AbstractProject<?, ?> project,
```

Comment:
In such case I'm OK with this line. There should be a more universal way to determine if a job has a workspace, but it's out of the scope of this PR